environments/india/app-processes.yml

```diff
@@ -10,24 +10,25 @@ celery_processes:
   celery1:
     beat: {}
     flower: {}
+  celery2,celery3:
```

Comment:
this is cool, i didn't know we could this. 👍

Reply:
Yeah, it's a new thing I added in https://github.com/dimagi/commcare-cloud/pull/3121 to make consistent setup across 2 or more machines really easy to maintain :)

Reply:
definitely easier and less lines :) thank you Danny